runtime/jvmti/jvmtiHook.c

```diff
@@ -618,6 +684,13 @@ processEvent(J9JVMTIEnv* j9env, jint event, J9HookRedirectorFunction redirectorF
 			} else {
 				return 0;
 			}
+
+		case J9JVMTI_EVENT_COM_IBM_VIRTUAL_THREAD_MOUNT:
+			return redirectorFunction(vmHook, J9HOOK_VIRTUAL_THREAD_MOUNT, jvmtiHookVirtualThreadMount, OMR_GET_CALLSITE(), j9env);
+
+		case J9JVMTI_EVENT_COM_IBM_VIRTUAL_THREAD_UNMOUNT:
+			return redirectorFunction(vmHook, J9HOOK_VIRTUAL_THREAD_UNMOUNT, jvmtiHookVirtualThreadUnmount, OMR_GET_CALLSITE(), j9env);
+
```

Comment:
This should be enabled for JDK19+.
```suggestion
#if JAVA_SPEC_VERSION >= 19
		case J9JVMTI_EVENT_COM_IBM_VIRTUAL_THREAD_MOUNT:
			return redirectorFunction(vmHook, J9HOOK_VIRTUAL_THREAD_MOUNT, jvmtiHookVirtualThreadMount, OMR_GET_CALLSITE(), j9env);

		case J9JVMTI_EVENT_COM_IBM_VIRTUAL_THREAD_UNMOUNT:
			return redirectorFunction(vmHook, J9HOOK_VIRTUAL_THREAD_UNMOUNT, jvmtiHookVirtualThreadUnmount, OMR_GET_CALLSITE(), j9env);
#endif /* JAVA_SPEC_VERSION >= 19 */
```